addons/l10n_lt/data/account_fiscal_position_template_data.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <!-- Account Fiscal Position Template -->
+    <record id="account_fiscal_position_template_eu" model="account.fiscal.position.template">
```

Comment:
Since these are auto applicable fiscal position, I would suggest to set `sequence` for them, may be 10 for `LT` and 20 for `EU`